src/App.js

```diff
@@ -18,7 +18,11 @@ function App() {
           <Route
             path="/"
             element={
-              games.length === 0 ? <GamesPage games={infoText} /> : <GamesPage games={games} />
+              games.length === 0 ? (
+                <GamesPage games={infoText} onDeleteGame={handleDeleteGame} />
```

Comment:
not sure if you need to have the delete function in it when only a text is visible without having saved something by myself or is it this example thing which is displaying and the user can therefore delete it?

Reply:
Im going to test it without the props, but i thought the GamesPage component need the probs anywere!